lib/Toolbar.js

```diff
@@ -56,7 +56,6 @@ export default class Toolbar extends Component {
         actions: PropTypes.arrayOf(PropTypes.shape({
             icon: PropTypes.string.isRequired,
             onPress: PropTypes.func,
-            counter: PropTypes.shape()
```

Comment:
Seems like it wasn't used anywhere in the file
On Jun 12, 2016 21:28, "Jiří Otáhal" <notifications@github.com> wrote:

> In lib/Toolbar.js
> <https://github.com/xotahal/react-native-material-ui/pull/8#discussion_r66725956>
> :
>
> > @@ -56,7 +56,6 @@ export default class Toolbar extends Component {
> >          actions: PropTypes.arrayOf(PropTypes.shape({
> >              icon: PropTypes.string.isRequired,
> >              onPress: PropTypes.func,
> > -            counter: PropTypes.shape()
>
> Why did you remove this line?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xotahal/react-native-material-ui/pull/8/files/4838dc7706101dd585ad8e3b761bec9def8458f5#r66725956>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABfmw0t-MXpfJv9_F4DJhgWk5uEKr4N7ks5qLF3rgaJpZM4IzJ1C>
> .
>